app/src/main/res/layout/lessons_locked_chapter_view.xml

```diff
@@ -35,7 +35,7 @@
         android:layout_gravity="center"
         android:layout_marginStart="10dp"
         android:fontFamily="sans-serif"
-        android:importantForAccessibility="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? View.IMPORTANT_FOR_ACCESSIBILITY_YES : View.IMPORTANT_FOR_ACCESSIBILITY_NO}"
```

Comment:
I think here the values should be `no` instead of yes because the value of `viewModel.chapterPlayState` for this file should be `ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES` which means that the condition will be false, therefore `no`

Reply:
Done.